src/app/pages/homepage/homepage.component.html

```diff
@@ -1,78 +1,248 @@
+<head>
```

Comment:
This leads to have a `<head>` section inside the `<body>` of the page, which is probably not what we want.

Reply:
Yes, I'm aware of this, but I wanted to use this font, so it is here temporarily.

Reply:
@vpchung You can add these lines to the `<head>` section of the file `src/index.html`

Reply:
Thanks for the suggestion, @rrchai !  I always forget how neat SCSS is.